package.json

```diff
@@ -305,25 +301,6 @@
         "title": "%azure-iot-toolkit.command.stopMonitorCustomEventHubEndpoint.title%",
         "category": "%azure-iot-toolkit.category%"
       },
-      {
```

Comment:
there are also some items in the command palette block to mark never shown.

Reply:
Removed